client/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { Stack, Card, CardContent, CardActionArea, Avatar, Typography, Rating, styled, Divider } from '@mui/material';
+import { LocationOn, Star } from '@mui/icons-material';
+import { useStyles } from './hooks';
+
+const ProfileAvatar = styled(Avatar)(({ theme }) => ({
+  width: 104,
+  height: 104,
+  background: theme.palette.grey[300],
+}));
+
+const StyledCard = styled(Card)(() => ({
+  boxShadow: '4px 4px 13px 7px rgba(217,217,217,0.26)',
+}));
+
+const Title = styled(Typography)(({ theme }) => ({
+  ...theme.typography.h5,
+  textAlign: 'center',
+  fontWeight: 600,
+}));
+
+const Status = styled(Typography)(({ theme }) => ({
+  ...theme.typography.caption,
+  textAlign: 'center',
+  fontSize: 14,
+  fontWeight: 500,
+  color: theme.palette.grey[500],
+}));
+
+const Description = styled(Typography)(({ theme }) => ({
+  ...theme.typography.body1,
+  textAlign: 'center',
+  fontSize: 16,
+  fontWeight: 400,
+}));
+
+const Location = styled(Typography)(({ theme }) => ({
+  ...theme.typography.body1,
+  color: theme.palette.grey[500],
+}));
+
+const ProfileCard = () => {
+  const classes = useStyles();
+  return (
+    <StyledCard variant="elevation" className={classes.root}>
+      <CardContent>
```

Comment:
```suggestion
 <CardContent>
    <CardActionArea component={Link} to={ `profile-details/${profile.id}` }
```
We can wrap the entire card with the `CardActionArea` and `Link` component as above